sample/build.gradle.kts

```diff
@@ -113,7 +113,7 @@ dependencies {
 
 if (useKsp) {
     ksp {
-        arg("boringyuri.type_adapter_factory", "boringyuri.sample.data.adapter.TypeAdapterFactory")
+        arg("boringyuri.type_adapter_factory", "boringyuri.sample.data.adapter.factory.TypeAdapterFactory")
```

Comment:
@finberg why does the package of the generated `TypeAdapterFactory` has to be changed though?

Reply:
@anton-novikau, we did not notice the bug because package of `TypeAdapterFactory` and packages of adaters it generated are the same. I have decided to change package of `TypeAdapterFactory`, probably it will help to reveal such code generation issues in the future.

Reply:
Yeah, you're right. When the factory and the adapter classes are in the same package they are implicitly imported and this hides the issue.

Okay, this makes sense. Approved ✅